cmd/mattermost/commands/plugin.go

```diff
@@ -172,12 +172,12 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 		return errors.New("Unable to list plugins. Error: " + appErr.Error())
 	}
 
-	CommandPrettyPrintln("Listing active plugins")
+	CommandPrettyPrintln("Listing enabled plugins")
 	for _, plugin := range pluginsResp.Active {
 		CommandPrettyPrintln(plugin.Manifest.Name + ", Version: " + plugin.Manifest.Version)
 	}
 
-	CommandPrettyPrintln("Listing inactive plugins")
+	CommandPrettyPrintln("Listing disenabled plugins")
```

Comment:
```suggestion
	CommandPrettyPrintln("Listing disabled plugins")
```

Reply:
:man_facepalming: thanks for catching this